_includes/analytics.html

```diff
@@ -1,5 +1,9 @@
-	<!-- Google Analytics -->
-
+<!-- Google Analytics -->
+	<!-- GA4 -->
+	<!-- Google tag (gtag.js) --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-HBYXWFP794"></script> 
+	<script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-HBYXWFP794'); </script>
```

Comment:
## Optional

We can condense some comments & move the `<script>` to its own line.

```suggestion
<!-- Google Analytics - GA4 -->
	<!-- Google tag (gtag.js) --> 
	<script async src="https://www.googletagmanager.com/gtag/js?id=G-HBYXWFP794"></script> 
	<script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-HBYXWFP794'); </script>
```

Reply:
LGTM!